Correct mOTLP API key privileges: APM application privilege required (#6515)
elastic/docs-content: solutions/observability/get-started/quickstart-elastic-cloud-otel-endpoint.md

  "name": "otlp-writer",
  "role_descriptors": {
    "otlp_writer": {
      "applications": [
        {
          "application": "apm",
          "resources": ["*"],
          "privileges": ["event:write"]
        }
      ]
    }
  }
}
```
:::{note}

> The API key authenticates the OTLP shipper to the {{motlp}}. The `event:write` privilege for the `apm` application is the minimum required to send data through the endpoint.

Reviewer: ```suggestion
The API key authenticates the OTLP shipper to the {{motlp}}.  To send data through the endpoint, it needs the `event:write` privilege for the `apm` application.
```
If accurate, might be simpler
